src/App.tsx

```diff
@@ -1,61 +1,153 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { User } from './components/types/User';
+import { TodoWithUser } from './components/types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+
+const preparedTodos: TodoWithUser[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: usersFromServer.find(({ id }) => id === todo.userId) || null,
+} as TodoWithUser));
+
+function findUserById(userId: number): User {
+  return usersFromServer.find(({ id }) => id === userId) as User;
```

Comment:
The same, if we don't find a user we return `undefind` but you told typescript that it will be user and you can miss an error